modules/suites/commerce/suite.bnd

```diff
@@ -0,0 +1,2 @@
+Liferay-Releng-Suite-Description: The Liferay Collaboration suite provides applications and features that help users share ideas, implement them, and produce terrific results. Its apps such as Message Boards and Blogs help users start discussions, ask questions, share ideas, and connect with other users. Wiki allows users to share in-depth information. And Alerts and Announcements facilitate broadcasting information and notifications.<br /><br />As users collaborate, they're sure to produce important files. The suite's Documents and Media Library allows users to store files, share them, and publish them. It not only works with Liferay's built-in repository but also allows you to integrate with external repositories, such CMIS repositories. Users can add and share all kinds of files, including documents, presentations, images, and videos. And the suite's Documents and Media app and Media Gallery help users display files the way they like them.<br /><br />As users engage in site activities, you'll want to recognize them and connect them with other users. The suite's social networking apps and features enable you to attribute social equity to active users automatically by increasing their rank on the site. You can even highlight user activities to show your site's liveliness. The Mentions feature facilitates notifying specific users in the context of site content, discussions, and comments. The social apps also help users share their thoughts on the your sites and other social sites, such as Facebook and Twitter. Combined with the suite's collaboration and document management features, the Collaboration suite's social features help develop awesome communities, foster high morale, and make work interesting.<br /><br />Apps and Features: Activities, Alerts, Announcements, Blogs, Bookmarks, Comments, Friends/My Sites/Portal/Site Members Directory, Documents and Media, Friend Requests, Image Uploader, Invite Members, Media Gallery, Message Boards, Mentions, Microblogs, Page Flags/Comments/Ratings, Quick Note, Requests, Social Activity, User/Group Statistics, Wiki.<br /><br />This suite may contain modules that depend on Foundation suite modules. This suite is available in the ${liferay.releng.app.title.prefix} bundle.
```

Comment:
@dsitu maybe we need to find a better commerce text, and ask Corbin

Reply:
oh yeah, opps ahha, I thought I had removed that description...